server/api/versions/index.get.ts

```diff
@@ -0,0 +1,28 @@
+import { fetchRApi } from "@/server/utils/rApi";
```

Comment:
Could you simplify this location of this file somewhat? I get the impression from the [docs](https://nuxt.com/docs/guide/directory-structure/server), that you could just put this code in `server/api/versions.ts` - is it because you will have some POSTs which will need to go into a `.post.ts` file? But even then, I don't think you need to make a separate folder for each path.

Reply:
OK, let's simplify the dir structure for now